continuous_integration/environment-3.10-jdk11-dev.yaml

```diff
@@ -31,14 +31,5 @@ dependencies:
 - tpot
 - tzlocal>=2.1
 - uvicorn>=0.11.3
-# fugue dependencies; remove when we conda install fugue
-- adagio
-- antlr4-python3-runtime<4.10
-- ciso8601
-- fs
-- pip
-- qpd
-- triad
 - pip:
-  # FIXME: tests are failing with fugue 0.7.0
-  - fugue[sql]>=0.5.3,<0.7.0
+  - fugue>=0.7.0
```

Comment:
A couple questions/comments here:

- I notice we're just pip installing `fugue` without the `sql` extra - is this now included in the core package?
- Is there any reason we need to bump the minimum version of Fugue here? Not sure if there's some changes between 0.5.3 and 0.7.0 that can't be rectified with compatibility code
- If we _do_ opt to bump our minimum Fugue version to 0.7.0, we can start using the conda package of Fugue instead of the pip package, and remove the `pip` section of the dev environments

Reply:
1. Yes in >=0.7.0 sql is built in, we don't need any extra
2. There are a lot of changes, but the most important thing is dependency. Because of the breaking change of antlr and duckdb, before 0.7.0 there has to be version caps, now in 0.7.0 we removed all version caps.
3. I feel conda is heavy, adding a package can make it more confused, but pip is light, also it doesn't expose other dependencies to the sensitive conda system. But if you prefer, we can change it to conda dependency.